storage/plugins/bibframe-linked-data/BibframeLinkedDataPlugin.php

```
@@ -844,6 +854,39 @@ private function fetchPublisher(int $id): ?array
+            'SELECT e.id, e.nome
+               FROM libri_editori le
+               JOIN editori e ON e.id = le.editore_id
+              WHERE le.libro_id = ?
+              ORDER BY le.ordine, e.nome'
+        );
+        if ($stmt !== false) {
+            $stmt->bind_param('i', $bookId);
+            $stmt->execute();
+            $result = $stmt->get_result();
+            if ($result instanceof \mysqli_result) {
+                while ($r = $result->fetch_assoc()) { $out[] = $r; }
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $r. Configured minimum length is 3. (undefined)

(ShortVariable)